packages/api-form-builder/src/plugins/crud/forms.models.ts

```diff
@@ -102,7 +102,8 @@ export const FormSubmissionCreateDataModel = withFields({
             ip: string({}),
             submittedOn: string({
                 value: new Date().toISOString()
-            })
+            }),
+            url: object()
```

Comment:
Can we be more specific here?

An example I pulled from another piece of code:


```ts
    reCaptcha: fields({
        value: {},
        instanceOf: withFields({
            enabled: boolean(),
            /**
             * Note: We've replaced "i18nString()" with "string()"
             */
            errorMessage: string({
                value: "Please verify that you are not a robot."
            })
        })()
    })
```